knora/dsplib/utils/validation.py

```diff
@@ -0,0 +1,22 @@
+import regex
```

Comment:
In the context of the Domain Model Initiative, we decided that  it would be good to collect all validation stuff that is distributed over all of dsp-tools into one place. This here is just the beginning. In the future, I will move all validation to here.